draft-ietf-quic-multipath.md

```diff
@@ -112,7 +112,7 @@ non-zero connection IDs in order to identify the path and respective
 packet number space.
 
 To add a new path to an existing QUIC connection with multipath support,
-a client starts a path validation on
+a client starts a path validation or sends a packet on
```

Comment:
Maybe be simply less concrete (as the next sentence already talks about address validation):
```suggestion
a client sends a packet with a connection ID belonging to a so-far unused Path ID on
```